project/src/main/java/com/google/sps/data/DataAccessor.java

```diff
@@ -47,26 +48,26 @@ public DataAccessor() {
   }
 
   /**
-  * @param userId the id of the user that we want to check the registration status about
-  * @return whether the user is registered to our system, by checking whether their id was
-  *     previously added to datastore.
-  */
+   * @param userId the id of the user that we want to check the registration
```

Comment:
nit: In comments, write "ID"

Reply:
(These are remains of a different PR, I restored the file to the version that's in the master)

Reply:
I still see "id" and not "ID" though.